ui/apps/platform/src/Containers/NetworkGraph/NetworkGraph.tsx

```diff
@@ -163,32 +162,8 @@ const model: Model = {
     ],
 };
 
-function findEntityById(
-    graphModel: Model,
-    id: string,
-    type: UrlDetailTypeValue | undefined = undefined
-): Record<string, any> | undefined {
-    if (
-        type === UrlDetailType.DEPLOYMENT ||
-        type === UrlDetailType.CIDR_BLOCK ||
-        type === UrlDetailType.EXTERNAL_ENTITIES
-    ) {
-        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-        // @ts-ignore
-        return graphModel.nodes?.find((node: { id: string }) => node.id === id);
-    }
-    if (type === UrlDetailType.NAMESPACE) {
-        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-        // @ts-ignore
-        return graphModel.groups?.find((group: { id: string }) => group.id === id);
-    }
-    let entity;
-    entity = graphModel.nodes?.find((node: { id: string }) => node.id === id);
-    if (!entity) {
-        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-        // @ts-ignore
-        entity = graphModel.groups?.find((group: { id: string }) => group.id === id);
-    }
+function findEntityById(graphModel: Model, id: string): NodeModel | undefined {
+    const entity = graphModel.nodes?.find((node: { id: string }) => node.id === id);
```

Comment:
We're only dealing with nodes, so we can just do the following

Reply:
Either I imagined it, or the demo app had a separate "groups" list. 😕

Reply:
groups are technically nodes (appended to the end of the nodes array), so that's where you might've gotten confused @vjwilson